src/components/carousel/_index.scss

```diff
@@ -0,0 +1,159 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-hover-before");
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-focus-before");
+			}
+		}
+
+		@include scss.component-with-states("carousel-buttons", [hover, focus]);
+	}
+
+	.slick-next {
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-next", [before]);
+	}
+
+	.slick-prev {
+		&:before {
+			font-family: "Arial", serif;
+			content: "<";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-previous", [before]);
+	}
+
+	.slick-prev.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+
+		@include scss.component-with-states("carousel-button-previous-disabled", [before]);
+	}
+
+	.slick-next.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+
+		@include scss.component-with-states("carousel-button-next-disabled", [before]);
+	}
+
+	[dir="rtl"] {
+		.slick-prev {
+			left: auto;
+
+			&:before {
+				content: ">";
+			}
+
+			@include scss.component-with-states("carousel-button-rtl-previous", [before]);
+		}
+
+		.slick-next {
+			right: auto;
+			@include scss.component-properties("carousel-button-next");
+
+			&:before {
+				content: "<";
+			}
+
+			@include scss.component-with-states("carousel-button-rtl-next", [before]);
+		}
+	}
+
+	/* Dots */
+	.slick-dotted.slick-slider {
+		@include scss.component-properties("carousel-slider-margin");
+	}
+
+	.slick-dots {
+		list-style: none;
+		display: block;
+
+		@include scss.component-properties("carousel-slider-dots");
+
+		li {
+			cursor: pointer;
+
+			@include scss.component-properties("carousel-slider-dots-list");
```

Comment:
Again, semantics on this one.  I think maybe these should be flipped.
```suggestion
		@include scss.component-properties("carousel-slider-dots-list");

		li {
			cursor: pointer;

			@include scss.component-properties("carousel-slider-dots");
```
The reason for that would be that the `.slick-dots` refers to the list, and the `li` is the list item (or the dot itself).

This may impact naming of the following as well:
`carousel-slider-dots-list-buttons`
`carousel-slider-dots-list-buttons-active`

Reply:
Or maybe refer to the list as `dots` and the `li` as `dot`:
```suggestion
		@include scss.component-properties("carousel-slider-dots");

		li {
			cursor: pointer;

			@include scss.component-properties("carousel-slider-dot");
```

Reply:
maybe instead of `dot` we call it `indicator` since it's potentially not always a 'dot'

Reply:
Would cause confusion with the API:  https://react-slick.neostack.com/docs/api/#dots.  There is also custom paging if they want to go a different route